csrc/abstract_tensor.h

```diff
@@ -1000,8 +1005,13 @@ struct TaggedAbstractTensor : AbstractTensorWithInfo<TagSetInfo<Tag>> {
             {tag_sets.begin(), tag_sets.end()}) {}
 
   const std::unordered_set<Tag>& getTags(int64_t i) const {
-    i = wrapDim(i, (int64_t)this->info.size());
-    return this->info[i].tags;
+    i = wrapDim(i, (int64_t)this->size());
+    return this->info_[i].tags;
+  }
+
+  std::unordered_set<Tag>& getTags(int64_t i) {
+    i = wrapDim(i, (int64_t)this->size());
+    return this->info_[i].tags;
```

Comment:
I added this non-const accessor so that we can manually insert and erase tags for a single axis.